TextAnalyzer/Main.cs

```diff
@@ -9,53 +9,40 @@ namespace TextAnalyzer
 {
     class Main
     {
-        int getResponse;
-
         //Delegate WriteMessege.
         Action<string> writeMessege = Writer.WriteMessege;
+        List<ICommand> optionsList = new List<ICommand>();
+
+        public Main()
+        {
+            optionsList.Add(new TemplateCommandOption());
+            optionsList.Add(new ExitCommand());
+
+            StartMenu();
+        }
 
         public void StartMenu()
         {
-            writeMessege("1.Type 1");
-            writeMessege("2.Type 2");
-            writeMessege("3.Type 3");
-            writeMessege("4.Type 4");
+            int i = 1;
+            foreach (var command in optionsList)
+            {
+                writeMessege($"{i++}. {command.Description}");
+            }
```

Comment:
var i = 1;
            foreach (var command in optionsList) writeMessege($"{i++}. {command.Description}");

Reply:
Adjusted, chociaż osobiście uważam że to jest jedynie kwestia konwencji i tego czy w danym projekcie ważniejsza jest czytelność czy zwięzłość.